tests/fakes/fake_blaise_service.py

```diff
@@ -69,6 +71,10 @@ def add_case_to_questionnaire(
             field_region=field_region,
             field_team=None,
             wave_com_dte=None,
+            rotational_knock_to_nudge_indicator=rotational_knock_to_nudge_indicator,
+            rotational_outcome_code=0
```

Comment:
This is hard to read, can some parens/indentation be added to make it clear?

Reply:
I think it looks ok

Reply:
Changed the way it works - sets it to 0 instead of none which also removes the need for setting the int as optional